assets/index.underscore

```diff
@@ -19,11 +19,8 @@
   <meta name="viewport" content="width=device-width,initial-scale=1">
   <meta http-equiv="Content-Language" content="en" />
   <link rel="shortcut icon" type="image/png" href="dashboard.assets/img/couchdb-logo.png"/>
```

Comment:
awesome, can we make the favicon configurable too? this was another thing we needed for pouchdb-fauxton